_posts/2023-11-23-announcing-dp-social-inbox.md

```diff
@@ -0,0 +1,101 @@
+
+With the Social Inbox enabled on your site, you can:
+
+* **Post to the Fediverse**: New articles can be posted to the Fediverse where followers can read, re-post, favorite, and send replies.
+
+* **Approve replies as comments**: When posts receive replies on the Fediverse, the website owner can approve them as reader comments that are then posted alongside the original article on the website.
+
+* **Streamline moderation**: Approve messages and follow requests one-by-one, automate the process on your own, or subscribe to existing block lists. (For now, Distributed.Press subscribes to [GardenFence](https://github.com/gardenfence/blocklist/tree/main) to automatically block servers that have shown to be sources of hateful rhetoric, harassment, and spam.) 
+
+* **Download your follower list**: If you would rather move your account to a different server, you can switch your Social Inbox instance very easily without needing to change the domain name for your site.
+
+To see which Fediverse clients are compatible with the Social Inbox, you can follow this [Github issue](https://github.com/hyphacoop/social.distributed.press/issues/24).
+
+## Sutty Integration
+
+If you aren't able or interested in coding your way to creating a static website or implementing the Social Inbox yourself, we have also integrated it into [Sutty](https://sutty.nl/en), a content management system for static Jekyll websites. All you need to do is set up a website on their platform and enable the Social Inbox in the configuration panel. Then, you will see a panel that will allow you to approve/deny replies as comments to your posts.
+
+All of Sutty's templates are compatible with the Social Inbox, in addition to the Magazine template, which is based on the layout of COMPOST magazine, our sister project. For a step-by-step guide on how to use the Social Inbox with Sutty, check out this [LINK:demo video].
+
+[LINK WHEN AVAILABLE: You can see how it looks by checking out our account on the Fediverse: @HyphaCoop@hypha.coop]
+
+## Getting Started
+
+There are a few ways to try out the Social Inbox:
+
+* **Sutty**: Create a static website on Sutty and use the Social Inbox panel on their platform admin panel: https://sutty.nl/en/
+
+* **Do it yourself**: Check out our documentation to see how you can set up the Social Inbox on your existing (or future) static website: https://docs.distributed.press/
+
+* **Hire us to help**: If you already have a static website publishing pipeline, but want to publish to IPFS and Hypercore and implement the Social Inbox, get in touch with us at hello@distributed.press and we can help you out. :)
+
+## What's Next
+
+Our next area of focus is to make it easier for people to discover websites and blogs on the dweb easier. Essentially, we're developing automation on top of Distributed Press that will announce newly published accounts in a format that can be subscribed to on the Fediverse. 
```

Comment:
i.e. It will announce newly published Mastodon accounts? Or posts from websites via Social Inbox? I don't quite get this feature. Is there something you can relate it to, like your email example above?

Reply:
Ah I can see how that's confusing... So my understanding is that Sutty will build a tool that will allow publishers to have their DP published sites be announced on an account on the Fediverse.